pkg/environment/utils.go

```diff
@@ -60,3 +75,31 @@ func GetResolvedChannel(channel, inputChan, outputChan string) (string, error) {
 		Message("channel " + channel + " not listed as an input or output").
 		Build()
 }
+
+func getChannelList(channelList string) []brokers.ChannelBroker {
```

Comment:
"getChannelList" makes me think this returns a slice containing only channel names. Could probably rename it, or write a description stating the exact return